polaris.shopify.com/pages/_app.tsx

```diff
@@ -13,7 +13,14 @@ import '../src/styles/globals.scss';
 const PUBLIC_GA_ID = 'UA-49178120-32';
 
 const gaPageView = (url: string) => {
-  window.gtag('config', PUBLIC_GA_ID, {page_path: url});
+  window.gtag('config', PUBLIC_GA_ID, {
+    page_path: url,
+    custom_map: {
```

Comment:
following the docs here https://developers.google.com/analytics/devguides/collection/gtagjs/custom-dims-mets